File: left/pyramid.ex
#Elixir
defmodule Pyramid do
  def main do
    IO.puts("Number:")
    {num, _} = IO.gets("") |> Integer.parse()
    draw(1, num)
  end

  defp draw(i, n) when i <= n do
    IO.puts(String.duplicate("*", i))
    draw(i + 1, n)
  end

  defp draw(_, _), do: :ok
end

Pyramid.main()

File: left/Erlang/pyramid.erl
-module(pyramid).
-export([main/1]).

main(_Args) ->
    io:format("Number: \n"),
    {ok, [Num]} = io:fread("", "~d"),
    print_pyramid(1, Num).

print_pyramid(I, N) when I =< N ->
    io:format("~s~n", [lists:duplicate(I, $*)]),
    print_pyramid(I + 1, N);
print_pyramid(_, _) -> ok.
